### PR TITLE
generic: backport pending driver for Winchip CH348 USB serial

### PR DESCRIPTION
**Pull request by dangowrt**
Import patch and package kernel module for Winchip CH348 USB-to-8x-UART chip.

Patch is still pending upstream: https://patchwork.kernel.org/project/linux-usb/patch/20240507131522.3546113-2-clabbe@baylibre.com/

### CODE REVIEW

**Comment by aparcar** [comment]
Yea testing already, thanks

**Comment by xdarklight** [comment]
@dangowrt and @aparcar could (at least) one of you please submit your `Tested-by` on the upstream patch?
I think (Johan) the usb-serial maintainer is currently busy or on leave. That said, I expect that once he returns that he'll merge the driver - and I hope that more `Tested-by`s will make his decision a bit easier.

As for the OpenWrt package: I'm a bit rusty there but it looks good, so thumbs up from my side.

**Comment by aparcar** [comment]
I tested this and it work fine. Also wrote a short email to the kernel mailing list, hope it went through.

**Review by aparcar — APPROVED** [review]

**Pull request merged**
